doc(1.8.0): add DR volumes support for v2 data engine (#977)
longhorn/website: content/docs/1.8.0/important-notes/_index.md

Reverting a volume to a snapshot created before Longhorn v1.7.0 is not supported due to an incorrect UserCreated flag set on the snapshot. The workaround is to back up the existing snapshots before upgrading to Longhorn v1.7.0 and restore them if needed. The bug is fixed in v1.7.0, and more information can be found [here](https://github.com/longhorn/longhorn/issues/9054).

### Disaster Recovery Volumes

> Disaster recovery volumes is supported since Longhorn v1.8.0.

Reviewer: ```suggestion
Disaster recovery volumes are supported from Longhorn v1.8.0.
```

Author: Updated, thank you!
